met-web/src/components/engagement/view/PreviewBanner.tsx

```diff
@@ -22,8 +22,9 @@ export const PreviewBanner = () => {
     const imageExists = !!savedEngagement.banner_url;
     const isScheduled = savedEngagement.status_id === EngagementStatus.Scheduled;
     const scheduledDate = new Date(savedEngagement.scheduled_date);
+    const scheduledTime = scheduledDate.toTimeString().split(':')[0] + ':' + scheduledDate.toTimeString().split(':')[1];
```

Comment:
you should be able to set your own date format using the date helper class (common/dateHelper) 
formatDate(savedEngagement.scheduled_date, 'MMM dd yyyy HH:mm') or whatever format you want!

Reply:
perfect